service/src/main/resources/api/service_openapi.yaml

```diff
@@ -157,6 +157,25 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/workspaces/v1/{workspaceId}/gcp/enablepet:
+    parameters:
+      - $ref: '#/components/parameters/WorkspaceId'
+    post:
+      summary: Allow a user to use their pet SA in a given workspace
```

Comment:
This is pretty terse. I don't believe we document what a pet service account is anywhere else in the API spec.

Reply:
Agreed. I think that's fine given that this is a workaround which we only expect the CLI to call, though